src/main/kotlin/no/nav/familie/ks/sak/api/EndretUtbetalingAndelController.kt

```diff
@@ -101,4 +106,12 @@ class EndretUtbetalingAndelController(
 
         return ResponseEntity.ok(Ressurs.success(behandlingService.lagBehandlingRespons(behandlingId = behandling.id)))
     }
+
+    @GetMapping(
+        path = ["/endret-utbetaling-begrunnelser"],
```

Comment:
Ikke noe dere har gjort altså, men jeg tenker på sikt så bør vi endre på navnet på feltet som inneholder VedtakBegrunnelser.

Per nå har vi et felt begrunnelse OG begrunnelser i endret utbetaling andel objektet, og det er forvirrende (begrunnelse er det bruker fyller ut, begrunnelser er det som vises i vedtaksbrevet 🤔).

Jeg lurer på om det er innafor å gjøre det allerede nå i denne pullrequesten så slipper man å lage teknisk gjeld oppgave på det.

Ideelt sett så burde begrunnelser vært omdøpt til vedtaksbegrunnelser, også endepunktet justert slik at det kommer tydelig fram at man henter vedtaksbegrunnelser for endret utbetalingsandeler og ikke begrunnelsene som er skrevet SB (som kan forvirres i dag gitt at endepunktet er `api/endretutbetalingandel/endret-utbetaling-begrunnelser`

Reply:
Jeg er enig at feltet begrunnelser burde endres på, men er samtidig ikke helt enig i at vi skal gjøre det i denne PRen. Spesielt siden denne oppgaven er fin å få ut og har ligget under arbeid lenge. Jeg kan lage et teknisk gjeld-kort på det så kan vi se på det senere 🙌

Reply:
Men jeg kan jo døpe om endepunktet altså, så får vi ikke den jobben i tillegg når vi skal endre begrunnelser om til noe annet ☺️

Reply:
Favro-kort opprettet: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-23583